src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.js

```diff
@@ -69,79 +55,81 @@ uiModules.get('apps/management')
 
       return indexPatterns.mapper.getFieldsForIndexPattern(pattern, {
         skipIndexPatternCache: true,
-      })
-      .catch((err) => {
-        // TODO: we should probably display a message of some kind
+      });
+    })
+    .then(
+      fields => {
```

Comment:
So it lines up with the errorback below. With this version of the implementation it's not necessary to use a single `.then()`, so I'll switch it to `.then().catch()`